pancancer_evaluation/data_models/tcga_data_model.py

```diff
@@ -232,6 +232,47 @@ def process_data_for_gene_id(self,
             self.y_test_df.status = np.random.permutation(
                 self.y_test_df.status.values)
 
+    def process_train_data_for_gene(self,
+                                    train_gene,
+                                    train_classification,
+                                    output_dir,
+                                    test_cancer_type=None,
+                                    shuffle_labels=False):
+        """
+        Prepare to train model on a given gene. Preparation of test data must
```

Comment:
Don't know if you want to add a bit more in the description about how the data is being prepared? Looks like you're splitting the data as well as making sure the training set includes gene expression data from only one gene and cancer type or one gene and any cancer type

Reply:
Yep, good call - this function is doing a few different things. I'll add some clarification.